confighelper/beaconbroadcaster.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2017-2018 Joel Allen Luellwitz and Andrew Klapp
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+""" Provides the broadcasting component of a a filesystem-based IPC mechanism."""
+
+__all__ = ['BeaconBroadcasterBroadcastException',
+           'BeaconBroadcasterInitException',
+           'BeaconBroadcaster']
+
+import datetime
+import logging
+import os
+import stat
+import tmpfs
+
+GROUP_RW_MODE = stat.S_IXUSR | stat.S_IRUSR | stat.S_IWUSR | stat.S_IXGRP | stat.S_IRGRP
+GROUP_RO_MODE = stat.S_IXUSR | stat.S_IRUSR | stat.S_IRGRP
+RW_MODE = stat.S_IXUSR | stat.S_IRUSR | stat.S_IWUSR
+
+BEACON_PATH = '/var/spool/'
```

Comment:
Again, I'd probably declare this closer to where you use it (and not as a constant) especially because it is only used in one method. I define PATHs at the top of other files because they typically rely on a PROGRAM_NAME constant (which is relatively likely to change).